### PR TITLE
Update default.env

### PR DESCRIPTION
**Pull request by ElGuillermo**
Typos and outdated infos removed

### CODE REVIEW

File: default.env [hunk 1]
```diff
@@ -122,24 +122,21 @@ HLL_HOST=
...
 HLL_PASSWORD=
 
 # -----------------------------
 # RCON Web Setttings
 # -----------------------------
 
-# This is the port where the Rcon Website will be accessible. Feel free to change. 
-# If you're running multiple instances you have to change it per instance
-# If you want to put a reverse proxy on your rcon (e.g. for HTTPS) and block the direct 
-# access then use: 127.0.0.1:8010
```
Comment (timraay) [inline default.env:135]: This is still possible, no?

File: default.env [hunk 2]
```diff
@@ -122,24 +122,21 @@ HLL_HOST=
...
+# This is the port where the CRCON website will be accessible. Feel free to change. 
+# If you're running multiple instances, you have to change it per instance
 RCONWEB_PORT=8010
 
 # Same as above but for https
 RCONWEB_PORT_HTTPS=9010
 
 # Ports for the public version (statistics)
-# If you don't want the public site write add "127.0.0.1:" in front of the port number, example: 127.0.0.1:7010
```
Comment (timraay) [inline default.env:142]: Even  though it no longer works, it still has a use case, hence I wouldn't consider this outdated. See #652

**Review by timraay — CHANGES_REQUESTED** [review]

File: default.env (hunk 2 above)
Comment (ElGuillermo) [reply default.env:142]: So we should find another way to present it, as we got two or three bug reports related to this the last days (?)

File: default.env (hunk 1 above)
Comment (ElGuillermo) [reply default.env:135]: Maybe. Not sure, as I do not use reversing. + See the other note below.

**Comment by cemathey** [comment]
This is something I'm very unfamiliar with this, I know some people use some of this stuff but when I set up URLs I do it differently.

File: default.env (hunk 1 above)
Comment (FlorianSW) [reply default.env:135]: This is absolutely possible and we should keep that comment :)

**Comment by ElGuillermo** [comment]
Let's stall this one...

**Pull request closed without merging**